src/gennylib/test/context_test.cpp

```diff
@@ -112,23 +114,30 @@ TEST_CASE("loads configuration okay") {
 
     SECTION("Valid YAML") {
         auto yaml = NodeSource(R"(
-SchemaVersion: 2018-07-01
-Clients:
-  Default:
-    URI: 'mongodb://localhost:27017'
-Actors:
-- Name: HelloWorld
-  Type: Nop
-  Count: 7
-        )",
+            SchemaVersion: 2018-07-01
```

Comment:
This looks like a non-equivalent change to me. In YAML indentation matters. Is there some property of this string literal that makes these equivalent?

Reply:
According to [YAML 1.2.2 spec](https://yaml.org/spec/1.2.2/#indentation-spaces): "Each node must be indented further than its parent node. All sibling nodes must use the exact same indentation level." So the absolute number of spaces doesn't matter, what matters is the relative number of spaces between YAML lines.

Reply:
I've been thinking about this, and I would really prefer it if we didn't do this. When I see indented strings like this in source code, it tends to be too jarring to the eye, and I think this is a valid concern, especially since this is purely a formatting, and therefore aesthetic, PR. Having dealt with too many bugs related to code formatting getting inside strings, my view of aesthetics here is based on that.

Turning single lines into multi-line strings is strictly worse. There have been cases where flow-based YAML strings have been converted into non-flow-based YAML, which changes the (fairly implicit) test coverage of the YAML parser here. These are not unambiguous improvements.

When it comes to formatting and linting changes, they should be AST equivalent to the code before them, and I think they should be unambiguously better.

If we already have multi-line YAML strings with code formatting in them, I would accept this change in those cases. But I don't think this is a super valuable use of time.

Could you please resubmit this PR without the non-string and variable name-related changes?